src/org/zaproxy/zap/extension/jruby/ZapAddOn.xml

```diff
@@ -1,11 +1,11 @@
 <zapaddon>
 	<name>Ruby scripting</name>
-	<version>5</version>
+	<version>6</version>
 	<status>beta</status>
 	<description>Allows Ruby to be used for ZAP scripting - templates included</description>
 	<author>ZAP Dev Team</author>
 	<url></url>
-	<changes>Add template for HTTP Sender script.</changes>
+	<changes>Update JRuby from 1.7.4 to 9.1.12.0</changes>
```

Comment:
`Add template for HTTP Sender script.` should not be deleted (this version/change was not released to marketplace yet).

Reply:
Ok, "changes" field restored.